configure.in

```diff
@@ -1145,6 +1142,9 @@ PKG_CHECK_MODULES([FREETYPE2],  [freetype2],
 PKG_CHECK_MODULES([TAGLIB],  [taglib >= 1.8],
   [INCLUDES="$INCLUDES $TAGLIB_CFLAGS"; LIBS="$LIBS $TAGLIB_LIBS"],
   AC_MSG_ERROR($missing_library))
+PKG_CHECK_MODULES([ICU_UC], [icu-uc >= 54.1],
```

Comment:
is the version 54.1 a hard requirement? Many older, but still used linux distros do not have such a new version.
e.g. ubuntu 12.04, which we still support, only has libicu48. Current ubuntu 14.10 has libicu52